README.md

```diff
@@ -315,6 +359,41 @@ You can access any URL or endpoint that references this address from your host.
 1. Make a change to component `X`, in its respective release (`X-release`).
 1. Run `make X-release compile images run` to build your changes and run them.
 
+#### Bumping a version in a release (or just make a change)
+
+For this example, lets suppose we want to update a release to a later tag.
+First of all checkout the desired commit:
+
+```
+host> cd src/loggregator/ && git checkout v81
```

Comment:
The submodule might itself have more submodules (in fact, it probably does unless it's loggregator, since they have separate `foo` and `foo-release` repos).  Basically, I'm saying that just checking out the top level submodule isn't going to get you the correct source code for `bosh create release`.